WS2012R2/lisa/remote-scripts/ica/Kdump_Config.sh

```diff
@@ -424,14 +424,11 @@ case $DISTRO in
     ;;
 esac
 
-grep -i vmlinuz $boot_filepath
-if [ $? -ne 0 ]; then
-    # Can not find kernel command in grub.cfg, need to update /etc/default/grub
-    default_grub="/etc/default/grub"
-    sed -i "s/crashkernel=\S*//g" $default_grub
-    sed -i "s/console=\S*//g" $default_grub
-    sed -i "/GRUB_CMDLINE_LINUX/ s/\"$/ crashkernel=$crashkernel\"/" $default_grub
-    grub2-mkconfig -o $boot_filepath
+# check if kernelopts available in grub2-editenv list, if yes, set kernelopts
+grub2-editenv - list | grep -i kernelopts
```

Comment:
I think it might be worth checking first if grub2-editenv command is present, and only at the end fallback to the sed option.

Reply:
Chris, thank you for your review and comment.
The command "grub2-editenv" is also present in rhel7 and fedora.
e.g. on Fedora,
"grub2-editenv -list" output saved_entry=Fedora (4.17.12-100.fc27.x86_64) 27 (Workstation Edition)
so think about using "grub2-editenv - list | grep -i kernelopts" to distinguish different methods or releases to replace directly check distribution. Is it reasonable? Thank you so much.